src/components/geocoder-panel.js

```diff
@@ -135,10 +135,14 @@ export default function GeocoderPanelFactory() {
         lon + GEOCODER_GEO_OFFSET,
         lat + GEOCODER_GEO_OFFSET
       ];
-      const {center, zoom} = geoViewport.viewport(bounds, [
+      const {zoom} = geoViewport.viewport(bounds, [
         this.props.mapState.width,
         this.props.mapState.height
       ]);
+      // center being calculated by geo-vieweport.viewport has a complex logic that
+      // projects and then unprojects the coordinates to determine the center
+      // Calculating a simple average instead as that is the expected behavior in most of cases
+      const center = [(bounds[0] + bounds[2]) / 2, (bounds[1] + bounds[3]) / 2];
```

Comment:
Per the comment above, `@mapbox/geo-viewport` has updated the way viewport is being determined, making the tests fail. It seems like this simplified implementation is what the expected behavior is, so I've revisited it accordingly. There doesn't seem to be a single place in which viewport is being calculated, which could be a potential improvement to encapsulate this type of logic. References:

- `viewport` function in v0.2.2 (pre-upgrade) - https://github.com/mapbox/geo-viewport/blob/v0.2.2/index.js#L24
- `viewport` function in the laster `master` (post-upgrade) - https://github.com/mapbox/geo-viewport/blob/master/index.js#L32